packages/terra-form-radio/src/Radio.scss

```diff
@@ -0,0 +1,207 @@
+:local {
+  .radio {
+    color: var(--terra-form-radio-font-color, rgb(0, 0, 0));
+    display: block;
+    font-family: inherit;
+    margin-left: var(--terra-form-radio-container-margin-left, none);
+    margin-right: var(--terra-form-radio-container-margin-right, 2rem);
+    margin-top: var(--terra-form-radio-container-margin-top, 0.625rem);
+    position: relative;
+
+    /* stylelint-disable selector-max-compound-selectors */
+    .label {
+      cursor: pointer;
+      display: inline-block;
+      font-size: var(--terra-form-radio-font-size, 1rem);
+      line-height: var(--terra-form-radio-line-height, 1.42);
+      padding-left: var(--terra-form-radio-padding-left, none);
+      position: relative;
+    }
+
+    .is-mobile-label {
```

Comment:
You should increase the specificity of this. Currently it's the same specificity as the .label styles above. If anyone ever comes and switches the order of these then the mobile styles would not override the normal style.